src/accelerate/data_loader.py

```diff
@@ -989,12 +1023,7 @@ def prepare_data_loader(
         )
 
     if isinstance(sampler, SeedableRandomSampler) and use_seedable_sampler:
```

Comment:
Note that here this is an unprepared dataloader, so it'd be better to have this be a `util` function called `get_sampler` (though I'm open to having this also be attached to the DL as a passthrough, could be handy with the trainer)

Reply:
it should be fine from here since the dataloader have been initialized above. I think you are thinking about [this part](https://github.com/huggingface/accelerate/pull/2723/files#diff-d9858283a2ced902233727f6fddde0a00831ad9a66a069e57231a5057d550bf6R918:~:text=if%20sampler_is_batch_sampler%3A-,if%20sampler_is_batch_sampler%3A,-sampler%20%3D%20getattr) where indeed this is unprepared

Reply:
I created the util function and called it in the different class for now since there is no difference. Do you want me to put in this function a specific file ?

Reply:
Ack sorry yeah I just noticed that line, my bad. Let's just do that here then it's fine!